official/resnet/resnet.py

```diff
@@ -608,3 +710,9 @@ def __init__(self, resnet_size_choices=None):
              'is not always compatible with CPU. If left unspecified, '
              'the data format will be chosen automatically based on '
              'whether TensorFlow was built for CPU or GPU.')
+
+    self.add_argument(
+        '-v', '--version', type=int, choices=[1, 2], dest="version",
+        metavar="", default=_DEFAULT_VERSION,
+        help="Version of resnet. (1 or 2)"
```

Comment:
Please describe more fully. Ditto for the comments above-- somewhere, we will need a more full description of what the differences are. Maybe just citations here, and further details in the README?